water_balance/tools/waterbalance.py

```diff
@@ -739,6 +739,8 @@ def get_aggregated_flows(self, link_ids, pump_ids, node_ids, model_part):
             ('leak', '_cum', np_2d_groundwater_node, 26, 1),
             ('rain', '_cum', np_1d_node, 27, 1),
             ('intercepted_volume', '_current', np_2d_node, 34, -1),
+            ('q_sss', '_cum', np_2d_node, 35, 1),
+            ('q_sss', '_cum', np_2d_node, 36, 1),
```

Comment:
('q_sss', '_cum_negative', np_2d_node, 36, -1),  # q_sss_out

Reply:
Deze is gefixed toch? met jouw pull-request op deze pull-request.